src/main/java/com/uber/jenkins/phabricator/conduit/DifferentialClient.java

```diff
@@ -121,4 +142,8 @@ public JSONObject postComment(String revisionID, String message) throws ConduitA
     protected JSONObject callConduit(String methodName, Map<String, String> params) throws ConduitAPIException, IOException {
         return conduit.perform(methodName, params);
     }
+
+    protected JSONObject callConduit(String methodName, JSONObject params) throws ConduitAPIException, IOException {
```

Comment:
Why the two different variants for sending data? Can we just use `Map` on `JSONObject` everywhere, or is there some advantage to having both?